bigframes/core/compile/compiler.py

```diff
@@ -155,10 +155,14 @@ def compile_rowcount(node: nodes.RowCountNode, ordered: bool = True):
 
 @_compile_node.register
 def compile_aggregate(node: nodes.AggregateNode, ordered: bool = True):
-    result = compile_unordered_ir(node.child).aggregate(
-        node.aggregations, node.by_column_ids, node.dropna
-    )
-    return result if ordered else result.to_unordered()
+    if ordered:
```

Comment:
child should be compiled in ordered mode if and only if we need the ordering information for the purpose of ordered aggregations.

Reply:
Fixed! Thanks!